src/augur-node/index.js

```diff
@@ -1,9 +1,10 @@
 "use strict";
+var ee = require("event-emitter");
```

Comment:
Suggest using `var EventEmitter` instead of `var ee`.  Even though this file is trivial it's best not to get in the habit of using overly-brief variable names.